src/assets/i18n/english.json

```diff
@@ -253,7 +253,8 @@
     "locked": "Locked - Click to unlock",
     "settings": "Open Settings",
     "unlocked": "Unlocked - Click to lock",
-    "unlock-to-move": "Locked - Unlock to allow moving of this overlay panel"
+    "unlock-to-move": "Locked - Unlock to allow moving of this overlay panel",
+    "delete-all": "Deletes all notifications"
```

Comment:
These need to be sorted alphabetically, otherwise the tslint will complain about it.

Reply:
I'm probably going to sound like a douche and nitpicker now, but `delete` should go before `disable`, because `e` is earlier in the alphabet than `i` 😬

Reply:
Not a douche at all,  just lack of attention to detail on my part :)

I normally trust linters over myself haha